### PR TITLE
MM-41101 - Enterprise Gate Playbook Key Metrics tab

### PR DESCRIPTION
**Pull request by cpoile**
#### Summary
- Basic key metrics tab
- Enterprise gating for the key metrics tab
- @abhijit-singh I was able to get the sizes correct

Looks like:
image

And the usage tab now looks like:
image

#### Ticket Link
- https://mattermost.atlassian.net/browse/MM-41101

#### Checklist

- [ ] ~~Telemetry updated~~
- [ ] ~~Gated by experimental feature flag~~
- [ ] ~~Unit tests updated~~

### CODE REVIEW

File: webapp/src/components/backstage/playbooks/playbook_key_metrics.tsx [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+};
+
+const PlaceholderRow = styled.div`
+    height: 260px;
+    margin: 32px 0;
+`;
+
+const OuterContainer = styled.div`
+    height: 100%;
+    background-color: rgba(var(--center-channel-color-rgb),0.04);
```
Comment (agarciamontoro) [inline webapp/src/components/backstage/playbooks/playbook_key_metrics.tsx:48]: Nit:
```suggestion
    background-color: rgba(var(--center-channel-color-rgb), 0.04);
```

File: webapp/src/components/backstage/playbooks/playbook_key_metrics.tsx [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+`;
+
+const InnerContainer = styled.div`
+    display: flex;
+    flex-direction: column;
+    padding: 20px;
+    max-width: 1120px;
+    margin: 0 auto;
+    font-family: 'Open Sans', sans-serif;
```
Comment (agarciamontoro) [inline webapp/src/components/backstage/playbooks/playbook_key_metrics.tsx:57]: Do we need the font-family?

**Review by agarciamontoro — APPROVED** [review]
Looks great! Again, just some non-blocking comments.

**Review by gbochora — APPROVED** [review]
Looks good!

File: webapp/src/components/backstage/playbooks/playbook_key_metrics.tsx (hunk 2 above)
Comment (cpoile) [reply webapp/src/components/backstage/playbooks/playbook_key_metrics.tsx:57]: Probably not -- I'll remove it.

File: webapp/src/components/backstage/playbooks/playbook_key_metrics.tsx (hunk 1 above)
Comment (cpoile) [reply webapp/src/components/backstage/playbooks/playbook_key_metrics.tsx:48]: 👍

File: webapp/src/components/backstage/playbooks/playbook_key_metrics.tsx [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+                    </PlaceholderRow>
+                </InnerContainer>
+            </OuterContainer>
+        );
+    }
+
+    return (
+        <OuterContainer>
+            <InnerContainer>
+                {'Key metrics tab content.'}
```
[the remark this answers is not in the document]
Comment (cpoile) [reply webapp/src/components/backstage/playbooks/playbook_key_metrics.tsx:35]: Yep, placeholder. The implementation will be merged before next release. 👍

**Pull request merged**